src/main/resources/db/changesets/20240522_addjobidandcreatedsnapshotidtorequest.yaml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+  - changeSet:
+      id: add_job_id_and_created_snapshot_id_to_request
+      author: srubenstein
+      changes:
+        - addColumn:
+            tableName: snapshot_request
+            columns:
+              - column:
+                  name: created_snapshot_id
+                  type: ${uuid_type}
+                  constraints:
+                    foreignKeyName: fk_created_snapshot_snapshot_request
+                    references: snapshot(id)
+                    deleteCascade: true
+              - column:
+                  name: job_id
```

Comment:
> I saw some others which used the column name flightid. I went back and forth, but ended up deciding I thought job_id was more readable and consistent with our naming for jobs, and more independent of whether we use stairway to manage jobs or not.

I think I'd follow existing convention here rather than going a different way. It's true that "job id" is a more general term than "flight id" but it'd be a big change to migrate away from stairway and we'd end up having to change a bunch of things at that time anyway. So I would use `flightid` here too.